src/app/about/about.component.html

```diff
@@ -56,7 +56,7 @@ <h2 class="bold">About Us</h2>
           <h5 class="bold">About YaCy</h5>
           <p>YaCy is a free search engine that anyone can use to build a search portal for their intranet or to help search
             the public internet. Read more about YaCy here -
-            <a class="page-link" href="https://{{yacyUrl}}/en/index.html" target="_blank">YaCy Decentralized Web Search</a>
+            <a class="page-link" href="https://yacy.net/en/index.html" target="_blank">YaCy Decentralized Web Search</a>
```

Comment:
Create a new variable, `yacySite=url.yacy.site` in ts file and and change variable in interpolated string i.e `{{yacySite}}` thats it.